<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>881</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="mainPage">
     <widget class="QTableView" name="searchTableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>720</width>
        <height>480</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>商品列表</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>30</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchDescription">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>55</number>
      </property>
      <property name="placeholderText">
       <string>请输入商品信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addToChartButton">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>90</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>添加购物车</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="number">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>数量</string>
      </property>
     </widget>
     <widget class="QPushButton" name="allProductButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>显示所有商品</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="orderPage">
     <widget class="QTableView" name="chartTableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>720</width>
        <height>480</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>购物车</string>
      </property>
     </widget>
     <widget class="QPushButton" name="orderButton">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>430</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>下单</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>160</y>
        <width>98</width>
        <height>69</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>更改高亮物品数量</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="modifyChartNum">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modifyChartButton">
         <property name="text">
          <string>更改</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="modifyProductPage">
     <widget class="QTableView" name="manageTableView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>720</width>
        <height>480</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addProductButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>20</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>添加商品</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>商品管理</string>
      </property>
     </widget>
     <widget class="QPushButton" name="discountButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>590</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>设置折扣</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>590</y>
        <width>240</width>
        <height>20</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QRadioButton" name="selectedProductButton">
         <property name="text">
          <string>选中商品</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="allBookButton">
         <property name="text">
          <string>图书</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="allFoodButton">
         <property name="text">
          <string>食品</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="allClothButton">
         <property name="text">
          <string>服饰</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>550</y>
        <width>182</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>折扣（0~1）</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="discount">
         <property name="maxLength">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="homePage">
     <widget class="QPushButton" name="modifypasswordButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chargeButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>充值</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>210</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>新密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="newPassword">
         <property name="maxLength">
          <number>12</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>330</y>
        <width>170</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>充值金额</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="chargeAmount">
         <property name="maxLength">
          <number>10</number>
         </property>
         <property name="cursorPosition">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>246</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Welcome!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nickname">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;nickname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>280</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remaining:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="remaining">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="payButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>420</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>支付</string>
      </property>
     </widget>
     <widget class="QTableView" name="orderTableView">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>190</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>150</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>订单</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>420</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>总计：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pay">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>20</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Debug User</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>70</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Debug Chart</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>110</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Debug Order</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelButton">
      <property name="geometry">
       <rect>
        <x>799</x>
        <y>420</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>取消订单</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>121</width>
       <height>449</height>
      </rect>
     </property>
     <attribute name="label">
      <string>首页</string>
     </attribute>
     <widget class="QPushButton" name="mainPageButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>首页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="orderPageButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>购物车</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetBackendButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>重置系统</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>121</width>
       <height>449</height>
      </rect>
     </property>
     <attribute name="label">
      <string>个人主页</string>
     </attribute>
     <widget class="QPushButton" name="myButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>我的</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
